toolchest_client/tools/tool.py

```diff
@@ -387,6 +397,9 @@ def run(self):
             and (self.group_paired_ends or self.num_input_files == 1) \
             and self._system_supports_parallel_execution()
 
+        if should_run_in_parallel:
+            self.is_async = False
```

Comment:
We may want to print a warning here in case someone is expecting an async parallel run.

Reply:
That's a good idea, will add